template/affine/index.yaml

```diff
@@ -82,11 +82,9 @@ spec:
       automountServiceAccountToken: false
       containers:
         - name: ${{ defaults.app_name }}
-          image: ghcr.io/toeverything/affine-graphql:stable-e043ecf
+          image: ghcr.io/toeverything/affine-graphql:stable-e98f035
```

Comment:
https://github.com/toeverything/AFFiNE/commit/e98f035 and releases https://github.com/toeverything/AFFiNE/releases/tag/v0.22.4